src/core/server/types.ts

```diff
@@ -22,3 +22,4 @@ export { PluginOpaqueId } from './plugins/types';
 export * from './saved_objects/types';
 export * from './ui_settings/types';
 export { EnvironmentMode, PackageInfo } from './config/types';
+export { CspOptions } from './csp';
```

Comment:
This is for usages from the client-side?

Reply:
Yep!